python-shell/README.md

```diff
@@ -27,13 +27,37 @@ mvn clean install -Pquick -Pstandalone
 Once this is running you can run the python example using the command:
 
 ```
-python3 python-shell/src/main/python/example.py
+python3 python-shell/src/gafferpy/example.py
 ```
 
 Alternatively if you have you own REST API running that is authenticated with
 PKI certificates then you can follow the pki example. Before using the example you
 will need to export your PKI certificate into a .pem file:
 
 ```
-python3 python-shell/src/main/python/examplePki.py
+python3 python-shell/src/gafferpy/examplePki.py
```

Comment:
Here we are assuming the user is running the command from the root of gaffer-tools. Below when the user runs setup.py we assume they are within the python directory. We need to be consistent. Perhaps at the top tell the user to cd into python-shell. Then all the commands could assume the they are being run from there